test/cassava/client_test.rb

```diff
@@ -96,6 +96,23 @@ def string_keys(hash)
           assert_equal [2,3].to_set, items.map { |x| x['a'] }.to_set
         end
 
+        context 'array arguments' do
+          should 'return values in between specified values' do
+            items = @client.select(:test).where(['a', 0, 2]).allow_filtering.execute
```

Comment:
What value does this have over doing e.g. `@client.select(:test).where('a > ? and a < ?', 0, 2)`? To me, the syntax is more difficult to read (it's not clear that the array means strict less than/strict greater than).

I'm not sure this is needed. Possibly the documentation could be improved to show that `<` and `>` are supported.